resources/packs/gcp/gcp.lr

```diff
@@ -1251,3 +1253,221 @@ private gcp.project.kmsService.keyring.cryptokey.version.externalProtectionLevel
   // Path to the external key material on the EKM when using EKM connection
   ekmConnectionKeyPath string
 }
+
+// GCP Dataproc Resources
+private gcp.project.dataprocService {
+  // Project ID
+  projectId string
+  // List of available regions
+  regions() []string
+  // List of Dataproc clusters in the current project
+  clusters() []gcp.project.dataprocService.cluster
+}
+
+// GCP Dataproc cluster
+private gcp.project.dataprocService.cluster @defaults("name") {
+  // Project ID
+  projectId string
+  // Cluster name
+  name string
+  // Cluster UUID
+  uuid string
+  // Cluster configuration
+  config gcp.project.dataprocService.cluster.config
+  // Labels associated with the cluster
+  labels map[string]string
+  // Contains cluster daemon metrics such as HDF and YARN stats
+  metrics dict
+  // Cluster status
+  status gcp.project.dataprocService.cluster.status
+  // Previous cluster status
+  statusHistory []gcp.project.dataprocService.cluster.status
+  // Virtual cluster config is used when creating a Dataproc cluster that does not directly control the underlying compute resources
```

Comment:
Do you want to add "Whether" to the beginning of this?

Reply:
Not sure it makes sense. This is not a flag but a whole bunch of settings that are only set whenever a virtual cluster is used

Reply:
Ah, I see. I was just trying to figure out why you have "is" in there. Now I get it. I recommend just removing the "is" to be consistent:

```suggestion
  // Virtual cluster config used when creating a Dataproc cluster that does not directly control the underlying compute resources
```